config.go

```diff
@@ -45,6 +45,8 @@ type Config struct {
 	LogOutput io.Writer
 	// HTTPClient is the http client
 	HTTPClient *http.Client
+	// Specify an Explicit Marathon Version
+	Version string
```

Comment:
I initially was planning to change the signature and implement something like what you're describing but ultimately wanted to avoid updating all of the test code. Although there is currently no other use for specifying the version in the main configuration I could potentially see other parts of the library benefiting from knowing the version of Marathon. It could even be populated programatically when instantiating the client by making a call to the `/v2/info` endpoint.